spec/controllers/account/projects_controller_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+RSpec.describe Account::ProjectsController, type: :controller do
+  render_views
+  let!(:user) { create(:user) }
+  let!(:workspace) { create(:workspace, user: user) }
+  let!(:project) { create(:project, workspace: workspace) }
+  let!(:project_valid_params) { { name: "test_project"} }
+  let!(:project_invalid_params) { { name: nil} }
+
+  before do
+    sign_in user
+  end
+
+  describe "GET #index in Account/Projects controller" do
+    it "returns http success" do
+      get :index, params: { workspace_id: workspace.to_param }
+      expect(response).to have_http_status(200)
+    end
+  end
+    describe "GET show in Account/Projects controller" do
+    it "must return show page" do
+      get :show, params: { workspace_id: workspace.to_param, id: project.to_param }
+      expect(response).to have_http_status(200)
+      expect(response).to render_template(:show)
+    end
+  end
+
+  describe "GET #new" do
+    it "returns http success" do
+      get :new, params: { workspace_id: workspace.to_param }
+      expect(response).to have_http_status(200)
+      expect(response).to render_template("account/projects/_form")
+    end
+  end
+
+  describe "POST create in Account/Projects controller" do
+    context "with walid attributes" do
+      it "must create new project" do
+        expect { post :create, params: { workspace_id: workspace.to_param, project: project_valid_params } }.to change(Project, :count).by(1)
+      end
+    end
+  
+    context "with inwalid attributes" do
+      it "must not create a project and render 'new' form" do
+        count_before = Project.count
+        post :create, params: { workspace_id: workspace.to_param, project: project_invalid_params }

```

Comment:
go w/ the same style,
`expect { post(:create, params: { workspace_id: workspace.to_param, project: project_invalid_params }) }.to change(Project, :count).by(0)`